### ISSUE TITLE
Skip to main content link is missing

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/earthworks#1431]
SiteImprove flags this as "Skip to main content is missing." It's not missing, but it's not first. "Skip to search" is first instead. See screenshot.

<img width="497" alt="Screenshot 2024-10-08 at 4 31 45 PM" src="https://github.com/user-attachments/assets/c6ea187e-1733-43ae-a691-06628ee1ce78">

--- 

**Guidance**
https://www.w3.org/WAI/WCAG22/Techniques/general/G1.html.

Given the guidance linked above (which suggests it should be first), I recommend that we reverse the order "Skip to main content" and then "Skip to search."

**Comment by jcoyne** [comment]
isn't the search the first element in the main content? Won't skipping to main require extra keystrokes?

**Comment by dbranchini** [comment]
It looks like there's an added link now, "Skip to first result," so the order should be "Skip to main content" (required), "Skip to first result," and then "Skip to search." We discussed this with SODA and at a DLUX team meeting. Here's the reasoning:

- We also want the tabbing order to be fixed - see #1259 (also discussed with SODA). Once that's fixed, then skip to main content goes to <main> and the first thing they'll encounter are facets.
- Skip to search is last because once you do a search (from home or the search page), you end up at the top of the page again, most likely want to explore the content before doing another search.

**Comment by dbranchini** [comment]
Also, note <main> is sufficient. We can eliminate the extra role and aria-label - see screenshot.

<img width="961" alt="Screenshot 2024-10-16 at 4 41 45 PM" src="https://github.com/user-attachments/assets/9127fe20-68bc-41ad-b0b7-005b46f71e53">

**Comment by corylown** [comment]
This will be fixed by updating Blacklight
